docs/content/en/docs/tasks/cluster/cluster-upgrades.md

```diff
@@ -36,15 +36,53 @@ EKS Anywhere `upgrade` also supports upgrading the following core components:
 * Cert-manager
 * Etcdadm CAPI provider
 * EKS Anywhere controllers and CRDs
-* Gitops controller (Flux) - this is an optional component, will be upgraded only if specified
+* GitOps controllers (Flux) - this is an optional component, will be upgraded only if specified
 
 The latest versions of these core EKS Anywhere components are embedded into a bundles manifest that the CLI uses to fetch the latest versions 
 and image builds needed for each component upgrade. 
 The command detects both component version changes and new builds of the same versioned component.
 If there is a new Kubernetes version that is going to get rolled out, the core components get upgraded before the Kubernetes
 version. 
 Irrespective of a Kubernetes version change, the upgrade command will always upgrade the internal EKS
-Anywhere components mentioned above to their latest available versions. These upgrade changes are backwards compatible.
+Anywhere components mentioned above to their latest available versions. Most upgrade changes are backwards compatible.
```

Comment:
"Most upgrade changes are backwards compatible." 
as a user, this raises more questions for me than it answers. Do we have a list of which are backward compatible and which are not? If this isn't something we clearly supply maybe remove this phrase

Reply:
how about "All upgrade changes are backwards compatible expect GitOps"?

Reply:
yeah that makes sense